apps/openmw/mwgui/itemwidget.cpp

```diff
@@ -16,7 +16,12 @@ namespace
     {
         if (count == 1)
             return "";
-        if (count > 9999)
+
+        if (count > 999999999)
+            return MyGUI::utility::toString(int(count/1000000000.f)) + "b";
+        else if (count > 999999)
+            return MyGUI::utility::toString(int(count/1000000.f)) + "m";
+        else if (count > 9999)
```

Comment:
I am not sure if we really need the float here. With huge amount of gold division result can get a rounding error and display wrong value.